cmd/repository.go

```diff
@@ -120,49 +118,49 @@ COMMANDS
 	rename
 	  Rename a previously installed repository from <old> to <new>. Only installed
 	  repositories can be renamed.
-	    $ func repository rename <name> <new name>
+	    $ {{.Prefix}}func repository rename <name> <new name>
 
 	remove
 	  Remove a repository by name.  Removes the repository from local storage
 	  entirely.  When in confirm mode (--confirm) it will confirm before
 	  deletion, but in regular mode this is done immediately, so please use
 	  caution, especially when using an altered repositories location
 	  (FUNC_REPOSITORIES environment variable or --repositories).
-	    $ func repository remove <name>
-
-EXAMPLES
-	o Run in confirmation mode (interactive prompts) using the --confirm flag
-	  $ func repository -c
-
-	o Add a repository and create a new Function using a template from it:
-	  $ func repository add boson https://github.com/boson-project/templates
-	  $ func repository list
-	  default
-	  boson
-	  $ func create -l go -t boson/hello-world
-	  ...
-
-	o List all repositories including the URL from which remotes were installed
-	  $ func repository list -v
-	  default
-	  boson	https://github.com/boson-project/templates
-
-	o Rename an installed repository
-	  $ func repository list
-	  default
-	  boson
-	  $ func repository rename boson boson-examples
-	  $ func repository list
-	  default
-	  boson-examples
-
-	o Remove an installed repository
-	  $ func repository list
-	  default
-	  boson-examples
-	  $ func repository remove boson-examples
-	  $ func repository list
-	  default
+	    $ {{.Prefix}}func repository remove <name>
+`,
+		Example: `
```

Comment:
Not a big deal, but I was trying to follow `git` output as closely as possible (since it is the one other CLI tool a developer is likely to be familiar with), even so far as tabs and spacing:
```
NAME
→func-<subcommand> - <short description>

SYNOPSIS
→func subcommand [-c|--confirm] [-v|--verbose]
<... repeated for each usage >

DESCRIPTION
→Primary Description
→→$ <example command embedded in text>

COMMANDS

→With no arguments, this help text is shown.  To <do the thing> with
→an interactive prompt, use the use the --confirm (-c) flag.
→  $ func <subcommand> -c

→<optional sub-sub command>
→  Sub-descriptions are indented with spaces rather than more tabs

→  An example sub-item
→    $ An example command

EXAMPLES
→o Example Command Description preceded by an empty-circle as bullet
→  $ func subcommand example indented with spaces
<repeated as necessary>
```
Notice how the "EXAMPLES" section immediately follows the high-level explanation of any subcommands; a section which may be omitted if there are none (in which case examples immediately follow the description).

Unfortunately Cobra has it's own idea about how the command help should be formatted, and using the "Examples" member of the struct defers to that choice.

I personally prefer the `git` format due to both clarity and likely developer familiarity, and to have the entire help text in a single variable/member for easier ongoing maintenance.  However, this is not something I think is particularly important if we'd rather use the Cobra style; just thought you'd like to know there was method to the madness!